context-free.md

```diff
@@ -2,21 +2,50 @@
 
 ##  Who is this programming language for?
 
+This language seems best suited to people who want to make somewhat intricate art.
+I imagine that it is mostly used by semi-technical enthusiasts, but could be used by others too.
 
 ## What is easy to do in this language? Why is it easy?
 
+I found recursive or fractal structures to be exceptionally easy to make because of the natural way in which rules
+can reference themselves.
+
+I also found transformations of scale, color, and position to be very easy to do thanks to the 'shape adjustment' system.
 
 ## What is hard to do in this language? Why is it hard?
 
+I found that this language makes it hard to tackle a particular shape. For example, when making the Koch Snowflake I
```

Comment:
I agree that it can be difficult to tackle a particular shape: it's easy create a 'generic' fractal (which is in a sense detailed in that there are fine characteristics), but not so easy to then tweak certain aspects of the fractal to produce a more specific shape.